homeassistant/components/airzone/select.py

```diff
@@ -153,6 +203,12 @@ def __init__(
             f"{self._attr_unique_id}_{system_zone_id}_{description.key}"
         )
         self.entity_description = description
+
+        options_dict = self.entity_description.options_fn(
+            self, description.options_dict
+        )
+        self._attr_options = list(options_dict)
```

Comment:
Are the options dynamic?

Reply:
Only for the zone modes, since the specific Airzone device may not have all HVAC modes.
For example, european devices lack the HEAT_COOL HVAC mode.

Reply:
No I rather mean, can they change on runtime?

Reply:
> No I rather mean, can they change on runtime?

Nope